### PR TITLE
add backend miss function

### PR DESCRIPTION
**Pull request by yajiedesign**

### CODE REVIEW

File: keras/backend/mxnet_backend.py [hunk 1]
```diff
@@ -1122,7 +1127,7 @@ def round(x):
     # Returns
         A tensor.
     """
-    raise NotImplementedError
+    return KerasSymbol(mx.sym.log(data=x.symbol))
```
Comment (piiswrong) [inline keras/backend/mxnet_backend.py]: wrong op?

File: keras/backend/mxnet_backend.py [hunk 2]
```diff
@@ -1146,6 +1151,7 @@ def pow(x, a):
     # Returns
         A tensor.
     """
+    return KerasSymbol(mx.sym.pow(base=x.symbol, exp=a.symbol))
```
Comment (piiswrong) [inline keras/backend/mxnet_backend.py]: does keras use number as a? do we need to test for number vs symbol?

File: keras/backend/mxnet_backend.py [hunk 3]
```diff
@@ -1154,7 +1160,7 @@ def clip(x, min_value, max_value):
     # Returns
         A tensor.
     """
-    raise NotImplementedError
+    return KerasSymbol(mx.sym.clip(src=x.symbol, a_min=min_value, a_max=max_value))
```
Comment (piiswrong) [inline keras/backend/mxnet_backend.py]: need to make sure min_value is always number

File: keras/backend/mxnet_backend.py [hunk 4]
```diff
@@ -1221,7 +1228,7 @@ def minimum(x, y):
     # Returns
         A tensor.
     """
-    raise NotImplementedError
+    return KerasSymbol(mx.sym.minimum(left=x.symbol, right=y.symbol))
```
Comment (piiswrong) [inline keras/backend/mxnet_backend.py]: lhs, rhs?

File: keras/backend/mxnet_backend.py [hunk 5]
```diff
@@ -1730,7 +1737,7 @@ def softplus(x):
     # Returns
         A tensor.
     """
-    raise NotImplementedError
+    return KerasSymbol(mx.sym.Activation(data=x.symbol, act_type='softrelu'))
```
Comment (yajiedesign) [inline keras/backend/mxnet_backend.py]: yes

File: keras/backend/mxnet_backend.py [hunk 6]
```diff
@@ -505,7 +490,12 @@ def ones(shape, dtype=None, name=None):
                [ 1.,  1.,  1.,  1.]], dtype=float32)
     ```
     """
-    raise NotImplementedError
+    if dtype is None:
+        dtype = floatx()
+    value = mx.nd.ones(shape, dtype=dtype)
+    if name is None:
+        name = _autogen_name('oneinit')
+    return KerasTensor(value, name)
```
Comment (piiswrong) [inline keras/backend/mxnet_backend.py]: I think this should be symbol, not tensor

Comment (yajiedesign) [inline keras/backend/mxnet_backend.py]: value  is Ndarray . the Ndarray  to KerasSymbol?

File: keras/backend/mxnet_backend.py (hunk 1 above)
Comment (yajiedesign) [inline keras/backend/mxnet_backend.py]: what wrong?

File: keras/backend/mxnet_backend.py [hunk 7]
```diff
@@ -917,7 +922,7 @@ def min(x, axis=None, keepdims=False):
     # Returns
         A tensor with miminum values of `x`.
     """
-    raise NotImplementedError
+    return KerasSymbol(mx.sym.min(data=x.symbol, axis=axis, keepdims=keepdims))
```
Comment (howard0su) [inline keras/backend/mxnet_backend.py]: i fixed in pr/6. you can leave it here.

File: keras/backend/mxnet_backend.py (hunk 1 above)
Comment (piiswrong) [inline keras/backend/mxnet_backend.py]: This is rounding to nearest int. Not log. I'll merge first. Please fix later

**Pull request merged**